ale/drivers/mex_drivers.py

```diff
@@ -584,3 +584,141 @@ def fikid(self):
               Naif ID code used in calculating focal length
             """
             return spice.bods2c(self.instrument_id)
+
+
+
+class MexSrcPds3NaifSpiceDriver(Framer, Pds3Label, NaifSpice, RadialDistortion, Driver):
+    """
+    Driver for a PDS3 Mars Express (Mex) High Resolution Stereo Camera (HRSC) - SRC image.
+    """
+
+    @property
+    def odtk(self):
+        """
+        The coefficients for the distortion model. No distortion model, so pass in all zeroes.
+
+        Returns
+        -------
+        : list
+          Radial distortion coefficients.
+        """
+        return [0.0, 0.0, 0.0]
+
+
+    @property
+    def ikid(self):
+        """
+        Returns the Naif ID code for HRSC SRC. 
+
+        Returns
+        -------
+        : int
+          Naif ID used to for indentifying the instrument in Spice kernels
+        """
+        return spice.bods2c("MEX_HRSC_SRC")
+
+
+    @property
+    def instrument_id(self):
+        """
+        Returns the short name of the instrument
+
+        MEX HRSC has nine different filters each with their own name.
+
+         Returns
+        -------
+        : str
+          Short name of the instrument
+        """
+        if(super().instrument_id != "HRSC"):
+            raise Exception ("Instrument ID is wrong.")
+        return self.label['DETECTOR_ID']
+
+
+    @property
+    def spacecraft_name(self):
+        """
+        Spacecraft name used in various SPICE calls to acquire
+        ephemeris data. MEX HRSC img PDS3 labels do not the have SPACECRAFT_NAME
+        keyword, so we override it here to use the label_pds3 property for
+        instrument_host_id
+
+        Returns
+        -------
+        : str
+          Spacecraft name
+        """
+        return self.instrument_host_id
+
+
+    @property
+    def focal2pixel_lines(self):
+        """
+        Expects fikid to be defined. This must be the integer Naif id code of
```

Comment:
Is this true?

Reply:
Nope, forgot to edit the docs. Thank you!